frontend/packages/local-storage-operator-plugin/src/components/local-volume-set/create-local-volume-set.tsx

```diff
@@ -85,13 +84,12 @@ const CreateLocalVolumeSet: React.FC = withHandlePromise<CreateLocalVolumeSetPro
     };
 
     if (showNodesList) {
-      const selectedNodesUID = getSelectedNodeUIDs(rows);
-      const selectedNodes = selectedNodesUID.map((uid) => rows[uid].props.data.metadata.name);
+      const selectedNodesNames = selectedNodes.map((node) => getName(node));
       requestData.spec.nodeSelector = {
         nodeSelectorTerms: [
           {
             matchExpressions: [
-              { key: 'kubernetes.io/hostname', operator: 'In', values: [...selectedNodes] },
+              { key: 'kubernetes.io/hostname', operator: 'In', values: [...selectedNodesNames] },
```

Comment:
```suggestion
              { key: 'kubernetes.io/hostname', operator: 'In', values: selectedNodesNames },
```
I don't see the need to make a copy.